scripts/ndk-make.sh

```diff
@@ -108,12 +108,14 @@ if test -z "$(find "$ANDROID_NDK_ROOT" -name libgcc.a -print -quit)"; then
     echo 'INPUT(-lunwind)' >"$TMPLIB/libgcc.a"
 fi
 
+RUSTUP_TOOLCHAIN=1.64.0
+
 if test -z $1 || test $1 = armeabi-v7a; then
     echo "-- cross compiling to armv7-linux-androideabi (arm) --"
     export CFLAGS=-D__ANDROID_API__=16
     TARGET_CC=armv7a-linux-androideabi16-clang \
     TARGET_AR=llvm-ar \
-    cargo +`cat rust-toolchain` rustc $RELEASEFLAG --target armv7-linux-androideabi -p deltachat_ffi -- -L "$TMPLIB"
+    cargo "+$RUSTUP_TOOLCHAIN" rustc $RELEASEFLAG --target armv7-linux-androideabi -p deltachat_ffi -- -L "$TMPLIB"
```

Comment:
I'm +/-0 on this PR, because the downside is that this will mean that we need to bump the Rust version in even more places every few months.

But maybe we can create one file `rust-toolchain` in the dc-android repo that contains the name of the currently used Rust version? This way, we'd only need to update it in one place.

E.g.:

```suggestion
    cargo +`cat ../../rust-toolchain` rustc $RELEASEFLAG --target armv7-linux-androideabi -p deltachat_ffi -- -L "$TMPLIB"
```